low-level/framework/utils/tool_factory.py

```diff
@@ -19,6 +19,8 @@
 
 from framework.utils.sysfs_interface import *
 from framework.utils.procfs_interface import *
+from framework.utils.dmidecode_interface import *
```

Comment:
This is unnecessory. why not just import the function and use it? 
Please avoid this and directly import the function in cpu_default_sensor.py

Reply:
This could be a generic way.
For eg : In health.py script, different class functions are called like Procfs, Dmidecode etc
This way we can import ToolFactory, and can get the utility instance
else, need to add imports for all files

Reply:
in that case only import required function. `*` import is not generally a good idea.

Reply:
okay. Importing the Classname , avoiding *